test/smoke/usm-globals-with-pragma/usm_globals_with_pragma.cpp

```diff
@@ -0,0 +1,52 @@
+#include <cstdio>
+
+#pragma omp requires unified_shared_memory
+
+//#pragma omp begin declare target
```

Comment:
Are the comments to indicate that this *would be the semantics* without USM pragma?

Reply:
Removed